definitions/checks/disk/available_space.rb

```diff
@@ -0,0 +1,22 @@
+module Checks
+  module Disk
+    class AvailableSpace < ForemanMaintain::Check
+      metadata do
+        label :available_space
+        description 'Check if root(/) partition has enough space to proceed with upgrade'
+        tags :pre_upgrade
```

Comment:
The original Issue requests this check also in health check which runs chcks tagged :default